### DISCUSSION TITLE
有人天翼网盘总失效吗

### DISCUSSION THREAD

**Discussion by yhh3344** [AlistGo/alist#738] [Q&A] [answered]
挂载了天翼网盘，24小时左右就会打不开文件，必须去天翼网盘账号列表重新保存才能进天翼网盘目录

错误
get empty login page

**Comment by xhofe** [comment, marked as answer]
新版本增加了由客户端协议实现 可以尝试使用：189CloudPC

**Reply by yhh3344** [reply]
好的，谢谢
